docs/devguide/modules-dev-guide.asciidoc

```diff
@@ -336,6 +354,88 @@ Note that you should follow the convention of naming of fields prefixed with the
 module and fileset name: `{module}.{fileset}.field`, e.g.
 `nginx.access.remote_ip`. Also, please review our <<event-conventions>>.
 
+[[ingest-json-entry-point-pipeline]]
+{ref}/conditionals-with-multiple-pipelines.html[In 6.6 and later], ingest pipelines can use the
+`pipeline` processor to delegate parts of the processings to other pipelines.
+
+[source,json]
+----
+{
+  "pipeline": {
+    "name": "name-of-another-pipeline"
+  }
+}
+----
+
+This can be useful if you want a fileset to ingest the same _logical_ information
+presented in different formats, e.g. csv vs. json versions of the same log files.
+Imagine an entry point ingest pipeline that detects the format of a log entry and then conditionally
+delegates further processing of that log entry, depending on the format, to another
+pipeline.
+
+[source,json]
+----
+{
+  "processors": [
+    {
+      "grok": {
+        "field": "message",
+        "patterns": [
+          "^%{CHAR:first_char}"
+        ],
+        "pattern_definitions": {
+          "CHAR": "."
+        }
+      }
+    },
+    {
+      "pipeline": {
+        "if": "ctx.first_char == '{'"
+        "name": "json-log-processing-pipeline"
+      }
+    },
+    {
+      "pipeline": {
+        "if": "ctx.first_char != '{'"
+        "name": "plain-log-processing-pipeline"
+      }
+    }
+  ]
+}
+----
+
+In order for the above pipeline to work, Filebeat must load the entry point pipeline
+as well as any sub-pipelines into Elasticsearch. You can tell Filebeat to do
+so by specifying all the necessary pipelines for the fileset in its `manifest.yml`
+file. The first pipeline in the list is considered to be the entry point pipeline.
+
+[source,yaml]
+----
+ingest_pipeline:
+  - ingest/main.json
+  - ingest/plain_logs.json
+  - ingest/json_logs.json
+----
+
+Additionally, any `pipeline` processors referencing one of the pipelines in the
```

Comment:
Then remove this sentence because you shouldn't need it because you show the correct syntax in the full example.